ckanext/cioos_theme/templates/header.html

```diff
@@ -87,8 +87,10 @@ <h2>{{ gns.site_description }}</h2>
     {% block header_site_navigation %}
     <nav>
       <ul>
-        {% block header_site_navigation_tabs %} {{ h.build_nav_main( ('search', _('Datasets')), ('organizations_index', _('Organizations')),
-        ('group_index', _('Groups')), ('home.about', _('About')) ) }} {% endblock %}
+        {% block header_site_navigation_tabs %}
+        <li><a href="{{ g.site_home_url }}">{{ _('HOME') }}</a></li>
+        {{ h.build_nav_main( ('search', _('Datasets')), ('organizations_index', _('Organizations')),
+        ('group_index', _('Groups')), ('home.about', _('About')), ('home.index', _('Data Catalogue')) ) }} {% endblock %}
```

Comment:
For home.index here - perhaps the hardcoded 'Data Catalogue' text could be replaced with a new element from the config to specify it?